message_ix_models/data/report/legacy/fil_files/AFR-HFC_fac.fil

```diff
@@ -0,0 +1,61 @@
+Variable,1990,1995,2000,2005,2010,2020,2030,2040,2050,2060,2070,2080,2090,2100,2110
```

Comment:
I would favour putting these under report/legacy/fil_files, until we have some documentation of their provenance, structure, code for handling them, etc.

Reply:
Ah, sorry—I misread the hierarchy! I thought they were in a directory /fil_files/ under the same parent directory as /report/. Please disregard.